<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogWalkThroughClass</class>
 <widget class="QDialog" name="DialogWalkThroughClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>386</width>
    <height>522</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::DefaultContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Walk through</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Camera path</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0" colspan="2">
         <widget class="QLabel" name="labelCharacterHeightFactor_4">
          <property name="text">
           <string>Load/Save camera path</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <spacer name="horizontalSpacer_9">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>10</width>
            <height>15</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="1">
         <layout class="QHBoxLayout" name="horizontalLayoutCameraPathButtons_3">
          <item>
           <widget class="QPushButton" name="importCameraPathButton">
            <property name="text">
             <string>Import</string>
            </property>
            <property name="default">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="exportCameraPathButton">
            <property name="text">
             <string>Export</string>
            </property>
            <property name="default">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_10">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>10</width>
              <height>15</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QCheckBox" name="checkBoxShowCameraPath">
            <property name="text">
             <string>Show path</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="2" column="0" colspan="2">
         <widget class="QLabel" name="labelCharacterHeightFactor_3">
          <property name="text">
           <string>Define a camera path</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <spacer name="horizontalSpacer_7">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>10</width>
            <height>15</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="3" column="1">
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QGridLayout" name="gridLayout_3">
            <item row="0" column="0">
             <widget class="QRadioButton" name="radioButtonFreeMode">
              <property name="text">
               <string>Free mode  </string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QPushButton" name="addKeyframeButton">
              <property name="text">
               <string>Add keyframe</string>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <spacer name="horizontalSpacer_8">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>50</width>
                <height>18</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="1" column="0" colspan="3">
             <widget class="QRadioButton" name="radioButtonWalkingMode">
              <property name="text">
               <string>Walking mode ('Ctrl + Left' click on the ground)</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayoutParameters">
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Preferred</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>18</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <layout class="QGridLayout" name="gridLayout">
              <item row="1" column="1">
               <widget class="QDoubleSpinBox" name="doubleSpinBoxCharacterDistanceFactor">
                <property name="decimals">
                 <number>2</number>
                </property>
                <property name="maximum">
                 <double>100.000000000000000</double>
                </property>
                <property name="singleStep">
                 <double>0.100000000000000</double>
                </property>
                <property name="value">
                 <double>1.900000000000000</double>
                </property>
               </widget>
              </item>
              <item row="0" column="0">
               <widget class="QLabel" name="labelCharacterHeight">
                <property name="text">
                 <string>Character height</string>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="labelCharacterDistanceToEye">
                <property name="text">
                 <string>Character distance to eye</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QDoubleSpinBox" name="doubleSpinBoxCharacterHeightFactor">
                <property name="decimals">
                 <number>2</number>
                </property>
                <property name="maximum">
                 <double>100.000000000000000</double>
                </property>
                <property name="singleStep">
                 <double>0.100000000000000</double>
                </property>
                <property name="value">
                 <double>0.200000000000000</double>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayoutCameraPathButtons">
            <item>
             <widget class="QPushButton" name="previousPositionButton">
              <property name="text">
               <string>Back</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="nextPositionButton">
              <property name="text">
               <string>Next</string>
              </property>
              <property name="default">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="removeLastPositionButton">
              <property name="text">
               <string>Undo</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="clearCameraPathButton">
              <property name="text">
               <string>Clear</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QSlider" name="horizontalSliderPreview">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="tickPosition">
             <enum>QSlider::TicksAbove</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="videoOutputGroupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Output</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="labelFrameRate">
          <property name="text">
           <string>Frame rate (fps)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBoxFPS">
          <property name="toolTip">
           <string>Number of frames per second</string>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="value">
           <number>25</number>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>13</width>
            <height>18</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="labelBitRate">
          <property name="text">
           <string>Bitrate (Mbps)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBoxBitRate">
          <property name="toolTip">
           <string>Bitrate (in Mbps)
The higher the better the quality (but the larger file size of the video)</string>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>1000</number>
          </property>
          <property name="singleStep">
           <number>1</number>
          </property>
          <property name="value">
           <number>10</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayoutOutputFile">
        <item>
         <widget class="QLabel" name="labelOutputFile">
          <property name="text">
           <string>Output file</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditOutputFile">
          <property name="toolTip">
           <string>The recording file name</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="browseButton">
          <property name="text">
           <string notr="true">...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="labelInterpolationSpeed">
          <property name="text">
           <string>Speed</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="doubleSpinBoxInterpolationSpeed">
          <property name="decimals">
           <number>1</number>
          </property>
          <property name="minimum">
           <double>1.000000000000000</double>
          </property>
          <property name="maximum">
           <double>10.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.100000000000000</double>
          </property>
          <property name="value">
           <double>1.000000000000000</double>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="previewButton">
          <property name="text">
           <string>Preview</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Minimum</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="recordButton">
          <property name="text">
           <string>Record</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBoxStartFromCurrentStep">
        <property name="text">
         <string>Start from current step</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>importCameraPathButton</tabstop>
  <tabstop>exportCameraPathButton</tabstop>
  <tabstop>checkBoxShowCameraPath</tabstop>
  <tabstop>radioButtonFreeMode</tabstop>
  <tabstop>addKeyframeButton</tabstop>
  <tabstop>radioButtonWalkingMode</tabstop>
  <tabstop>doubleSpinBoxCharacterHeightFactor</tabstop>
  <tabstop>doubleSpinBoxCharacterDistanceFactor</tabstop>
  <tabstop>previousPositionButton</tabstop>
  <tabstop>nextPositionButton</tabstop>
  <tabstop>removeLastPositionButton</tabstop>
  <tabstop>clearCameraPathButton</tabstop>
  <tabstop>horizontalSliderPreview</tabstop>
  <tabstop>spinBoxFPS</tabstop>
  <tabstop>spinBoxBitRate</tabstop>
  <tabstop>lineEditOutputFile</tabstop>
  <tabstop>browseButton</tabstop>
  <tabstop>doubleSpinBoxInterpolationSpeed</tabstop>
  <tabstop>previewButton</tabstop>
  <tabstop>recordButton</tabstop>
  <tabstop>checkBoxStartFromCurrentStep</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
